spec/services/remove_status_service_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe RemoveStatusService do
+  subject { RemoveStatusService.new }
+
+  let!(:alice)  { Fabricate(:account, username: 'alice') }
```

Comment:
I do not think `alice` and `jeff` should have fixed usernames. Why not make use of Fabrication and Faker?